.github/workflows/pyscal.yml

```diff
@@ -12,35 +12,34 @@ on:
       - published
   schedule:
     # Run CI every night and check that tests are working with latest dependencies
-    - cron:  '0 0 * * *'
+    - cron: "0 0 * * *"
 
 jobs:
-
   pyscal:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
         include:
-          - os: 'ubuntu-latest'
-            python-version: '3.8'
-          - os: 'ubuntu-latest'
-            python-version: '3.9'
-          - os: 'macos-latest'
-            python-version: '3.9'
-          - os: 'windows-latest'
-            python-version: '3.9'
+          - os: "ubuntu-latest"
+            python-version: "3.8"
+          - os: "ubuntu-latest"
+            python-version: "3.9"
+          - os: "macos-latest"
+            python-version: "3.9"
+          - os: "windows-latest"
+            python-version: "3.9"
 
     steps:
       - name: Checkout commit locally
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
 
       - name: Checkout tags
         # This seems necessary for setuptools_scm to be able to infer
         # the correct version.
         run: git fetch --unshallow --tags
```

Comment:
You could replace this by adding 

```yaml
with:
  fetch-depth: 0
```
 in the checkout action step if you wanted to, but it doesn't really make much difference (fetch-depth may actually be a bit heavier).

Reply:
Thanks @mferrera I was distracted and forgot about this PR.